holidays/calendars/islamic.py

```
@@ -4264,3 +4264,76 @@ def tasua_dates(self, year: int) -> Iterable[tuple[date, bool]]:
+        2016: (JUL, 6),
+        2017: (JUN, 25),
+        2018: (JUN, 15),
+        2019: (JUN, 5),
+        2020: (MAY, 24),
+        2021: (MAY, 13),
+        2022: (MAY, 3),
+        2023: (APR, 22),
+        2024: (APR, 10),
+        2025: (MAR, 31),
+        2026: (MAR, 21),
+    }
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)